src/test/java/it/gov/pagopa/arc/controller/TransactionsControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package it.gov.pagopa.arc.controller;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import it.gov.pagopa.arc.dto.TransactionDTO;
+import it.gov.pagopa.arc.service.TransactionsService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import utils.TestUtils;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(value = {
+        TransactionsController.class
+})
+class TransactionsControllerTest {
+    @Autowired
+    private ObjectMapper objectMapper;
+    @Autowired
+    private MockMvc mockMvc;
+    @MockBean
+    private TransactionsService transactionsServiceMock;
+
+    @Test
+    void getTransactionsList() throws Exception {
```

Comment:
let's use the naming convention defined in the Handbook https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/962232530/HB+Dev+-+Spring+Boot+Unit+Testing+Best+Practice#%40Test

`givenXXXWhenYYYThenZZZ`